src/linear_algebra/tensor_product_basis.lean

```diff
@@ -27,6 +27,15 @@ finsupp.basis_single_one.map
     (finsupp_tensor_finsupp R _ _ _ _).trans $
     finsupp.lcongr (equiv.refl _) (tensor_product.lid R R)).symm
 
+@[simp]
+lemma basis.tensor_product_apply (b : basis ι R M) (c : basis κ R N) (i : ι) (j : κ) :
+  (basis.tensor_product b c) (i, j) = b i ⊗ₜ c j :=
+by { dunfold basis.tensor_product, simp }
+
+lemma basis.tensor_product_apply' (b : basis ι R M) (c : basis κ R N) (i : ι × κ) :
+  (basis.tensor_product b c) i = b i.1 ⊗ₜ c i.2 :=
+by { dunfold basis.tensor_product, simp }
```

Comment:
```suggestion
  basis.tensor_product b c i = b i.1 ⊗ₜ c i.2 :=
by simp [basis.tensor_product]
```

Reply:
I might even be tempted to prove this like this:
```lean
lemma basis.tensor_product_apply' (b : basis ι R M) (c : basis κ R N) (ij : ι × κ) :
  basis.tensor_product b c ij = b ij.1 ⊗ₜ c ij.2 :=
by { obtain ⟨i, j⟩ := ij, exact b.tensor_product_apply c i j, }
```
so that it's using the API provided by `basis.tensor_product_apply` rather than the implementation.